src/main/java/com/knighttodo/knighttodo/service/impl/TodoBlockServiceImpl.java

```diff
@@ -19,54 +16,40 @@
 public class TodoBlockServiceImpl implements TodoBlockService {
 
     private final TodoBlockGateway todoBlockGateway;
-    private final TodoBlockMapper todoBlockMapper;
 
     @Override
     public TodoBlockVO save(TodoBlockVO todoBlockVO) {
-        TodoBlock todoBlock = todoBlockGateway.save(todoBlockMapper.toTodoBlock(todoBlockVO));
-
-        return todoBlockMapper.toTodoBlockVO(todoBlock);
+        return todoBlockGateway.save(todoBlockVO);
     }
 
     @Override
     public List<TodoBlockVO> findAll() {
-        return todoBlockGateway.findAll()
-            .stream()
-            .map(todoBlockMapper::toTodoBlockVO)
-            .collect(Collectors.toList());
+        return todoBlockGateway.findAll();
     }
 
     @Override
-    public TodoBlockVO findById(long todoBlockId) {
-        Optional<TodoBlock> result = todoBlockGateway.findById(todoBlockId);
-        TodoBlockVO todoBlock;
+    public TodoBlockVO findById(long id) {
+        Optional<TodoBlockVO> result = todoBlockGateway.findById(id);
 
         if (result.isPresent()) {
-            todoBlock = todoBlockMapper.toTodoBlockVO(result.get());
-        } else {
-            throw new RuntimeException("Did not find TodoBlock id - " + todoBlockId);
+            return result.get();
         }
-
-        return todoBlock;
+        throw new RuntimeException("Did not find TodoBlock id - " + id);
```

Comment:
perhaps we'll move it to constants later of maybe do something with localization.

Reply:
Ok)